src/main/java/org/broadinstitute/hellbender/tools/reference/CheckReferenceCompatibility.java

```diff
@@ -0,0 +1,284 @@
+            referenceCompatibilities.add(new CompatibilityRecord(referenceDictPath, "All sequence names and lengths match in the sequence dictionaries. Since the MD5s are lacking, we can't confirm there aren't mismatching bases in the references."));
+        }
+        else if(compatibilityStatus.equals(SequenceDictionaryUtils.SequenceDictionaryCompatibility.SUPERSET)){
+            referenceCompatibilities.add(new CompatibilityRecord(referenceDictPath, String.format("All sequence names and lengths present in the sequence dictionaries match, but %s is a subset of %s. Missing sequence(s): %s. Since the MD5s are lacking, we can't confirm there aren't mismatching bases in the references.", dictionaryName, referenceDictName, getMissingSequencesIfSubset(referenceDict))));
+        }
+        else{
+            referenceCompatibilities.add(new CompatibilityRecord(referenceDictPath, String.format("Not compatible. Status: %s. Run CompareReferences tool for more information on reference differences.", compatibilityStatus)));
+        }
+    }
+
+    private List<String> getMissingSequencesIfSubset(SAMSequenceDictionary referenceDict){
+        Set<String> commonSequences = SequenceDictionaryUtils.getCommonContigsByName(dictionaryToCompare, referenceDict);
+        List<String> missingSequences = SequenceDictionaryUtils.getContigNamesList(referenceDict);
+        missingSequences.removeAll(commonSequences);
+
+        return missingSequences;
+    }
+
+    @Override
+    public Object onTraversalSuccess() {
+        return super.onTraversalSuccess();
+    }
+
+    @Override
+    public void closeTool() {
+        super.closeTool();
+    }
+
+    /**
+     * Minimal class representing a record in the compatibility output table.
+     */
+    private static class CompatibilityRecord {
+        GATKPath ref;
+        String compatibilityOutput;
+
+        CompatibilityRecord(GATKPath ref, String compatibility){
+            this.ref = ref;
+            compatibilityOutput = compatibility;
+        }
+
+        public String getRefAsString(){
+            return ref.toPath().getFileName().toString();
+        }
+
+        public String getCompatibility(){
+            return compatibilityOutput;
+        }
+    }
+
+    /**
+     * TableWriter to format and write the table output.
+     */
+    public static class CheckReferenceCompatibilityTableWriter extends TableWriter<CompatibilityRecord> {
+
+        public CheckReferenceCompatibilityTableWriter(final Path table, TableColumnCollection columns) throws IOException {
+            super(table, columns);
+        }
+
+        public CheckReferenceCompatibilityTableWriter(final Writer writer, TableColumnCollection columns) throws IOException {
+            super(writer, columns);
+        }
+
+        @Override
+        protected void composeLine(final CompatibilityRecord record, final DataLine dataLine){
+            dataLine.set("Reference", record.getRefAsString())
```

Comment:
Thinking on this. I think this tool might be a little too verbose as it is. I think we should add the option to the user to have a more succinct summary of referenc - status. I think the best way to do that would be to change this object to have 3 rows, Reference, Status, Summary. Where we put all of the human readable summary text into the third category and that will allow users in the future to more easily manipulate the output of the tool down the road if they run it with many references.